packages/amplify_datastore_plugin_interface/test/testData/Comment.dart

```diff
@@ -79,13 +90,13 @@ class Comment extends Model {
 
   Comment.fromJson(Map<String, dynamic> json)
       : id = json['id'],
-        post = json['post'] != null
+        _post = json['post'] != null
             ? Post.fromJson(new Map<String, dynamic>.from(json['post']))
             : null,
-        content = json['content'];
+        _content = json['content'];
```

Comment:
Should we validate `json['content']` here to ensure it's non-null? Asking because that it looks like class `Comment` ensures `_content` cannot be assigned `null`, except this constructor. Thinking it may be better to keep the behavior consistent. (same question to other `fromJson` methods in other classes)

Reply:
_content can be null as it is a nullable type.   Typically the field should not be null, but in fromJson we don't know if all fields are going to be set as it might be a nested model in which only the id field will be set.